Gopkg.toml

```diff
@@ -39,8 +39,8 @@ required = [
 # This controls when we upgrade apis independently of Serving.
 [[override]]
   name = "github.com/knative/pkg"
```

Comment:
Is there a reason we don't just bring this to current HEAD?

Reply:
it is a moving target

Reply:
I am matching what eventing is using.

Reply:
you convinced me, updating...